pages/[id]/edit.js

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
+import React from "react";
+import FormEdit from "@/components/FormEdit";
+import { useRouter } from "next/router";
+
+const StyledHeadline = styled.h1`
+  text-align: center;
+`;
+
+export default function EditActivityPage({ onEditActivity, activities }) {
+  const router = useRouter();
+  const { id } = router.query;
+
+  return (
+    <>
+      <StyledHeadline>Edit Activity</StyledHeadline>
+      <FormEdit
```

Comment:
Looks good, but just as a suggestion: It might be more handy to reuse the form you're using for create.